src/main/java/io/jenkins/plugins/synopsys/security/scan/bridge/BridgeDownloadManager.java

```diff
@@ -143,6 +145,7 @@ public String downloadVersionFileFromArtifactory(String directoryUrl) {
             }
         } catch (IOException | InterruptedException e) {
             logger.error("An exception occurred while downloading 'versions.txt': " + e.getMessage());
+            Thread.currentThread().interrupt();
```

Comment:
Need to check the unit tests as there are code coverage warnings from jenkins.
There are more warnings similar to this.